src/distributing-policies.md

```diff
@@ -34,5 +35,5 @@ $ wasm-to-oci push pod-runtime-class-policy.wasm \
               <oci-registry>/chimera-policies/pod-runtime-class-policy:v0.0.1
 ```
 
-The policy can then be referenced from the Chimera admission controller as
+The policy can then be referenced from the Chimera admission server as
```

Comment:
I **think** @ereslibre is changing `controller` with `server` because the actual code now is inside of the `policy-server` repository, instead of the `chimera-admission-controller`.

I would instead change `Chimera admission controller` to `Chimera policy server`.

Opinions?

Reply:
`Chimera policy server` sounds good to me.